src/algorithm/model.hpp

```diff
@@ -144,6 +144,31 @@ namespace pinocchio
                     ModelTpl<Scalar,Options,JointCollectionTpl> & reduced_model,
                     GeometryModel & reduced_geom_model);
 
+  /**
+   *
+   *  \brief Build a reduced model and a rededuced geometry model  from a given input model, a given input geometry model and a list of joint to lock.
+   *
+   *  \param[in] model the input model to reduce.
+   *  \param[in] list_of_geom_models the input geometry model to reduce (example: visual_model, collision_model).
+   *  \param[in] list_of_joints_to_lock list of joints to lock in the input model.
+   *  \param[in] reference_configuration reference configuration.
+   *  \param[out] reduced_model the reduced model.
+   *  \param[out] list_of_reduced_geom_model the list of reduced geometry models.
+   *
+   *  \remarks All the joints that have been set to be fixed in the new reduced_model now appear in the kinematic tree as a Frame as FIXED_JOINT.  
+   *
+   */
+  template <typename Scalar, int Options,
+            template <typename, int> class JointCollectionTpl,
+            typename ConfigVectorType>
+  void buildReducedModel(
+      const ModelTpl<Scalar, Options, JointCollectionTpl> &model,
+      const std::vector<GeometryModel> &list_of_geom_models,
+      const std::vector<JointIndex> &list_of_joints_to_lock,
+      const Eigen::MatrixBase<ConfigVectorType> &reference_configuration,
+      ModelTpl<Scalar, Options, JointCollectionTpl> &reduced_model,
+      std::vector<std::reference_wrapper<GeometryModel>> &list_of_reduced_geom_models);
```

Comment:
You should avoid `>>` for C++98

Reply:
Wow, I didn't know that, could you propose a solution in that case?

BTW, it could be useful to [enforce compliance via the compiler](https://stackoverflow.com/a/12606757) to avoid having to review this quirks.

Reply:
The CI is not checking this compliance, we may think to add it, but I don't have time for that.
you should add a space `> >`

Reply:
> you should add a space > >

I didn't realize that was the problem haha

Reply:
should be fixed now